src/EFCore.Relational/Query/SqlExpressions/SqlFunctionExpression.cs

```diff
@@ -12,8 +12,24 @@
 
 namespace Microsoft.EntityFrameworkCore.Query.SqlExpressions
 {
+    /// <summary>
+    ///     <para>
+    ///         An expression that represents SQL FUNCTION.
```

Comment:
```suggestion
    ///         An expression that represents a function call in SQL.
```

(because this doesn't really get generated as FUNCTION?)